projects/vdk-control-cli/src/vdk/internal/control/command_groups/job/secrets.py

```diff
@@ -0,0 +1,324 @@
+# Copyright 2021-2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import _io
+import json
+import logging
+from enum import Enum
+from enum import unique
+from typing import Any
+from typing import Dict
+from typing import Tuple
+from typing import Type
+
+import click
+from vdk.internal.control.configuration.defaults_config import load_default_team_name
+from vdk.internal.control.exception.vdk_exception import VDKException
+from vdk.internal.control.rest_lib.factory import ApiClientFactory
+from vdk.internal.control.rest_lib.rest_client_errors import ApiClientErrorDecorator
+from vdk.internal.control.utils import cli_utils
+from vdk.internal.control.utils import output_printer
+from vdk.internal.control.utils.output_printer import OutputFormat
+
+log = logging.getLogger(__name__)
+
+
+@unique
+class SecretOperation(Enum):
+    """
+    An enum used to store the types of secrets operations
+    """
+
+    SET = "set"
+    GET = "get"
+
+
+class JobSecrets:
+    def __init__(
+        self,
+        rest_api_url: str,
+        job_name: str,
+        team: str,
+        output_format: OutputFormat,
+    ):
+        self.__secrets_api = ApiClientFactory(rest_api_url).get_secrets_api()
+        self.__output = output_format
+        self.__printer = output_printer.create_printer(output_format)
+        self.__job_name = job_name
+        self.__team = team
+        self.__deployment_id = "TODO"
+
+    def __get_all_remote_secrets(self) -> Dict[str, Any]:
+        return self.__secrets_api.data_job_secrets_read(
+            team_name=self.__team,
+            job_name=self.__job_name,
+            deployment_id=self.__deployment_id,
+        )
+
+    def __list_secrets(self, remote_secrets: Dict[str, Any]) -> None:
+        self.__printer.print_dict(remote_secrets)
+
+    @staticmethod
+    def _to_bool(value: str) -> bool:
+        if value == "true" or value == "True":
```

Comment:
would you not use to lowercase and only do one comparison?

Reply:
fixed.